README.md

```diff
@@ -22,4 +22,4 @@ Learn the implemented App and the example and reimplement it with Redux having t
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - This task does not have tests yet!
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_redux-list-of-posts/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<kostiukmkalne>.github.io/react_redux-list-of-posts/) and add it to the PR description.
```

Comment:
```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://kostiukmkalne.github.io/react_redux-list-of-posts/) and add it to the PR description.
```
https://kostiukmkalne.github.io/react_redux-list-of-posts/

provide this link in the PR description

Reply:
Got it!
Thanks

Reply:
https://.github.io/react_redux-list-of-posts/